src/app/tests/TestICDManager.cpp

```diff
@@ -198,7 +198,7 @@ class TestICDManager
 
         // Set New durations for test case
         uint32_t oldActiveModeDuration_ms = icdConfigData.GetActiveModeDurationMs();
-        icdConfigData.SetModeDurations(0, icdConfigData.GetIdleModeDurationSec());
+        icdConfigData.SetModeDurations(MakeOptional(static_cast<uint32_t>(0)), Optional<uint32_t>::Missing());
```

Comment:
`Optional<uint32_t>::Missing()` can just be chip::NullOptional IIRC.

Reply:
you recall correctly. Updated the function call.